### PR TITLE
First round of fixes post QDMC

### PR DESCRIPTION
**Pull request by boyercb**

### CODE REVIEW

File: master_check.do [hunk 1]
```diff
@@ -283,7 +279,7 @@ if ${run_outliers} {
 /* <============= HFC 12. Check for and output field comments =============> */
 
 if ${run_field_comments} {
-  ipacheckcomment ${fieldcomments}, id(${id}) ///
+  ipacheckcomment finalnotes, id(${id}) ///
```
Comment (boyercb) [inline master_check.do]: Was this change intentional @rsandino

File: master_check.do [hunk 2]
```diff
@@ -14,14 +14,14 @@ ipadoheader, version(15.0)
...
-ipacheckimport using "04_checks/01_inputs/hfc_inputs.xlsm"
+ipacheckimport using "../04_checks/01_inputs/hfc_inputs.xlsm"
 
 
 /* =============================================================== 
    ==================== Replace existing files  ================== 
    =============================================================== */
 
-foreach file in "${outfile}" "${enumdb}" "${textaudit}" "${researchdb}" "${bcfile}"{
+foreach file in "${outfile}" "${enumdb}" "${researchdb}" "${bcfile}" "${progreport}" "${dupfile}"{
```
Comment (boyercb) [inline master_check.do]: I think these are old too @rsandino

File: master_check.do [hunk 3]
```diff
@@ -296,11 +292,10 @@ if ${run_field_comments} {
 /* <=============== HFC 13. Output summaries for text audits ==============> */
 
 if ${run_text_audits} {
-  ipachecktextaudit ${textaudit} using "${infile}",  ///
-    saving("${textauditdb}")  ///
+  ipachecktextaudit ${textaudit} using "${textauditdb}",  ///
     media("${sctomedia}") ///
     enumerator(${enum}) ///
-    keepvars(${keep13})
+    keepvars(${keep15})
```
Comment (boyercb) [inline master_check.do]: Should this be 15 or 13, it's changed back and forth a few times in these changes?

File: master_check.do (hunk 2 above)
Comment (rsandino) [reply master_check.do]: progreport and dupfile do need to be in there so they don't overwrite but the removal of textaudit was unintentional. I'll fix that now

File: master_check.do (hunk 1 above)
Comment (rsandino) [reply master_check.do]: No I'll fix it now

File: master_check.do (hunk 3 above)
Comment (rsandino) [reply master_check.do]: I don't see either when I run macrolist?

Comment (rsandino) [reply master_check.do]: Never mind that was my mistake. Fixed!
